src/main/java/org/spongepowered/common/config/SpongeConfig.java

```diff
@@ -399,9 +399,18 @@ public TimingsCategory getTimings() {
         @Setting(comment = "A mapping from unqualified command alias to plugin id of the plugin that should handle a certain command")
         private Map<String, String> aliases = new HashMap<>();
 
+        @Setting(value = "multi-world-patches", comment = "Patches the specified commands to respect the world of the sender instead of applying the "
+                + "changes on the main overworld.")
+        private Map<String, Boolean> patches = new HashMap<>();
```

Comment:
I'd recommend using `multi-world-command-patches`, which means this field and the associated getter should probably be named `multiWorldCommandPatches`/`getMultiWorldCommandPatches` to match.